packages/amplify-codegen/src/codegen-config/AmplifyCodeGenConfig.js

```diff
@@ -61,8 +61,8 @@ class AmplifyCodeGenConfig {
     this.gqlConfig.config.projects = projects;
   }
   removeProject(projectName) {
-    if (Object.keys(this.gqlConfig.config).includes(projectName)) {
-      delete this.gqlConfig.config[projectName];
+    if (Object.keys(this.gqlConfig.getProjects()).includes(projectName)) {
+      delete this.gqlConfig.config.projects[projectName];
```

Comment:
This doesn't delete the graphql config yml file though right?

Reply:
Thats fine. We do remove the project configuration.